src/nft/utils/roundAndPluralize.ts

```diff
@@ -0,0 +1,5 @@
+export const roundAndPluralize = (i: number, word: string) => {
```

Comment:
This doesn't work with internationalization.

Reply:
We'll be tackling internationalization in a later sprint. When we do, will we not be able to have any kind of generalized behaviour helper function here or will we just need to use a conditional for the full word?

Reply:
you will not be able to use a helper function, you'll need to use the i18n libraries (lingui) to deal with pluralization.